app/reducers/events.js

```diff
@@ -248,6 +264,11 @@ export const selectEvents = createSelector(
   (eventsById, eventIds) => eventIds.map(id => transformEvent(eventsById[id]))
 );
 
+export const selectPreviousEvents = createSelector(
+    selectEvents,
+    events => events.filter(event => !event.isUsersUpcoming)
```

Comment:
Since all (other) events will have `isUserUpcoming: undefined`, so this could just use `events.filter(event => event.isUsersUpcoming===false)`